cobc/parser.y

```diff
@@ -12101,6 +12101,10 @@ display_body:
 	cb_emit_command_line ($1);
   }
 | screen_or_device_display _common_exception_phrases
+| _with CONVERSION screen_or_device_display _common_exception_phrases
+  {
+	CB_PENDING ("WITH CONVERSION");
```

Comment:
Now after seeing the testcase output (which showed itself to be reasonable by highlighting the c/s issue): 

Please change that to `CB_PENDING ("DISPLAY WITH CONVERSION");` and the testcase accordingly (as the current one looks a bit ugly).

This PR can be seen as "solved" then (sorry for not posting the complete relevant comments altogether, but I hope there was a bit of learning involved in which case the time was not lost).

Reply:
> Please change that to `CB_PENDING ("DISPLAY WITH CONVERSION");` and the testcase accordingly (as the current one looks a bit ugly).

Done. However I'm not sure what you mean about the test case looking "a bit ugly" : could you clarify, so that I can make better testcases in the future ?

> sorry for not posting the complete relevant comments altogether, but I hope there was a bit of learning involved

Yes there was ;)